loopd/daemon.go

```diff
@@ -102,8 +107,8 @@ func (d *Daemon) Run() error {
 	// With our client created, let's now finish setting up and start our
 	// RPC server.
 	serverOpts := []grpc.ServerOption{}
-	grpcServer := grpc.NewServer(serverOpts...)
-	looprpc.RegisterSwapClientServer(grpcServer, d)
+	d.grpcServer = grpc.NewServer(serverOpts...)
```

Comment:
could this be moved to `New`? That way we don't need to check whether it's nil in `stop`

Reply:
There won't be a `grpcServer` if we start the daemon as a subserver. So we need the nil check anyway.